client/my-sites/site-settings/test/form-google-analytics.jsx

```diff
@@ -33,24 +41,25 @@ const props = {
 	eventTracker: ( x ) => x,
 	uniqueEventTracker: ( x ) => x,
 	fields: {},
+	setDisplayForm: () => {},
 };
 
 describe( 'GoogleAnalyticsForm basic tests', () => {
 	test( 'simple form should not blow up and have proper CSS class', () => {
-		const comp = shallow( <GoogleAnalyticsSimpleForm { ...props } /> );
-		expect( comp.find( '#analytics' ) ).toHaveLength( 1 );
+		const { container } = render( <GoogleAnalyticsSimpleForm { ...props } /> );
+		expect( container.querySelectorAll( '#analytics' ) ).toHaveLength( 1 );
```

Comment:
Could we instead do a query by role `form`? Currently, the `<form>` element has the more abstract role of `section` (or none in some browsers), however, if we add an accessible name, it should be queryable by the role of `form`? See https://www.w3.org/TR/html-aria/#el-form for more info.

Reply:
Good call. I'm by no means an expert in a11y. Would https://github.com/Automattic/wp-calypso/pull/65226/commits/d4683a79ad24474fe6168c5e52c16f36fcfec782 work?